diff -ruN SOAP-Lite-1.13.orig/lib/SOAP/Transport/HTTP.pm SOAP-Lite-1.13/lib/SOAP/Transport/HTTP.pm
--- SOAP-Lite-1.13.orig/lib/SOAP/Transport/HTTP.pm	2014-12-30 16:48:04.000000000 +0100
+++ SOAP-Lite-1.13/lib/SOAP/Transport/HTTP.pm	2015-03-19 01:53:24.669732344 +0100
@@ -36,14 +36,15 @@
     return if $_patched;
     BEGIN { local ($^W) = 0; }
     {
+        no warnings "redefine";
         local $^W = 0;
         sub LWP::UserAgent::redirect_ok;
         *LWP::UserAgent::redirect_ok = sub { 1 }
     }
     {
-
         package
             LWP::Protocol;
+        no warnings "redefine";
         local $^W = 0;
         my $collect = \&collect;    # store original
         *collect = sub {
