diff -ruN SOAP-Lite-1.13.orig/lib/SOAP/Lite.pm SOAP-Lite-1.13/lib/SOAP/Lite.pm
--- SOAP-Lite-1.13.orig/lib/SOAP/Lite.pm	2014-12-30 16:48:04.000000000 +0100
+++ SOAP-Lite-1.13/lib/SOAP/Lite.pm	2015-03-19 01:51:24.951309918 +0100
@@ -4367,6 +4367,7 @@
 the following code:
 
     $client->on_action( sub { join '/', @_ } );
+
 =item on_fault(callback)
 
     $client->on_fault(sub { popup_dialog($_[1]) });
