--- SOAP-Lite-0.710.10/lib/SOAP/Transport/HTTP.pm.orig	2009-10-09 14:36:37.000000000 +0200
+++ SOAP-Lite-0.710.10/lib/SOAP/Transport/HTTP.pm	2009-09-30 20:34:59.000000000 +0200
@@ -38,13 +38,14 @@
     return if $_patched;
     BEGIN { local ($^W) = 0; }
     {
+        no warnings "redefine";
         local $^W = 0;
         sub LWP::UserAgent::redirect_ok;
         *LWP::UserAgent::redirect_ok = sub { 1 }
     }
     {
-
         package LWP::Protocol;
+        no warnings "redefine";
         local $^W = 0;
         my $collect = \&collect;    # store original
         *collect = sub {
